### PR TITLE
pulseaudio: enable dbus dependency

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **pulseaudio/***

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by Croydon — APPROVED** [review]

**Comment by ericLemanissier** [comment]
I tested it locally with `-o pulseaudio:with_dbus=True`, but it failed with:
```
/usr/bin/ld: ./.libs/module-equalizer-sink.a(module_equalizer_sink_la-module-equalizer-sink.o): in function `sink_input_pop_cb':
module-equalizer-sink.c:(.text+0x3d89): undefined reference to `fftwf_free'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x3da9): undefined reference to `fftwf_malloc'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x4177): undefined reference to `fftwf_execute_dft_r2c'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x42b8): undefined reference to `fftwf_execute_dft_c2r'
/usr/bin/ld: ./.libs/module-equalizer-sink.a(module_equalizer_sink_la-module-equalizer-sink.o): in function `module_equalizer_sink_LTX_pa__done':
module-equalizer-sink.c:(.text+0x49fa): undefined reference to `fftwf_destroy_plan'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x4a06): undefined reference to `fftwf_destroy_plan'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x4a0f): undefined reference to `fftwf_free'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x4a39): undefined reference to `fftwf_free'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x4a4e): undefined reference to `fftwf_free'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x4a7b): undefined reference to `fftwf_free'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x4a84): undefined reference to `fftwf_free'
/usr/bin/ld: ./.libs/module-equalizer-sink.a(module_equalizer_sink_la-module-equalizer-sink.o):module-equalizer-sink.c:(.text+0x4ab7): more undefined references to `fftwf_free' follow
/usr/bin/ld: ./.libs/module-equalizer-sink.a(module_equalizer_sink_la-module-equalizer-sink.o): in function `module_equalizer_sink_LTX_pa__init':
module-equalizer-sink.c:(.text+0x4f9b): undefined reference to `fftwf_malloc'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x4fe9): undefined reference to `fftwf_malloc'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x501b): undefined reference to `fftwf_malloc'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x50e0): undefined reference to `fftwf_malloc'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x518f): undefined reference to `fftwf_malloc'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x51be): undefined reference to `fftwf_plan_dft_r2c_1d'
/usr/bin/ld: module-equalizer-sink.c:(.text+0x51db): undefined reference to `fftwf_plan_dft_c2r_1d'
collect2: error: ld returned 1 exit status
```
Which happens probably because [`fftw` has a default precison `double`](https://github.com/conan-io/conan-center-index/blob/master/recipes/fftw/all/conanfile.py#L25), not single. I'm not sure why this error does not happen when dbus is disbaled

**Review by a4z — DISMISSED** [review]

**Review by prince-chrismc — DISMISSED** [review]

**Review by intelligide — DISMISSED** [review]

**Comment by ericLemanissier** [comment]
Ok so the compilation error I mentioned earlier happened only when dbus is enabled, simply because fftw is not used at all when dbus is disabled.  The only file to use fftw is this one: [module-equalizer-sink.c](https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/blob/master/src/modules/module-equalizer-sink.c). as can be seen, it uses ffwf, so it's compatible only with fft single precision

I reflected this in the recipe.

**Review by prince-chrismc — APPROVED** [review]

**Review by intelligide — APPROVED** [review]

**Review by SSE4 — DISMISSED** [review]

File: recipes/pulseaudio/all/conanfile.py [hunk 1]
```diff
@@ -65,15 +72,14 @@ def requirements(self):
             self.requires("libalsa/1.2.4")
         if self.options.with_glib:
             self.requires("glib/2.64.0")
```
Comment (SpaceIm) [inline recipes/pulseaudio/all/conanfile.py:74]: missed that: this version doesn't exist in CCI.

**Review by SpaceIm — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

File: recipes/pulseaudio/all/conanfile.py (hunk 1 above)
Comment (ericLemanissier) [reply recipes/pulseaudio/all/conanfile.py]: I have no idea how you are able to spot this kind of mistake!

Comment (SpaceIm) [reply recipes/pulseaudio/all/conanfile.py]: d820481ef14b1cd2a16ff4e7660deb5f

Actually, I try to check dependencies versions, and I was pretty sure that glib 2.64.0 was old.

Comment (prince-chrismc) [reply recipes/pulseaudio/all/conanfile.py]: Thank you Space for all your hard work... its noticed

**Review by SSE4 — APPROVED** [review]

**Pull request merged**